cinema/src/main/java/com/greg/cinema/entity/MovieEntity.java

```diff
@@ -0,0 +1,49 @@
+package com.greg.cinema.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "movies")
+public class MovieEntity {
+	
+	@Id
+	@GeneratedValue
+	Integer id;
+	
+	@Column(nullable=false, length=64)
```

Comment:
no nie wiem czy bym tak narzucał ograniczenie na długość https://collider.com/9-movies-that-have-very-very-very-very-very-long-titles/

Reply:
;)